src/databricks/labs/remorph/snow/snowflake.py

```diff
@@ -228,6 +228,12 @@ class Tokenizer(Snowflake.Tokenizer):
             r"(?i)var\s+\w+\s+=\s+\w+?": TokenType.VAR,
         }
 
+        SINGLE_TOKENS = {
+            **Snowflake.Tokenizer.SINGLE_TOKENS,
+            "&": TokenType.PARAMETER,  # https://docs.snowflake.com/en/user-guide/snowsql-use#substituting-variables-in-a-session
```

Comment:
This is incorrect. This breaks existing snowflake tokens, as sqlglot already maps parameter to `$` - `"$": TokenType.PARAMETER`.

This is purely `snowcli` variable substitution concept. *We need a new token type here.*

Reply:
for snowflake both '$' and '&' will be converted to "$" in databricks.